internal/backend/storage/fs/loader.go

```diff
@@ -37,20 +37,26 @@ func (l loader) Init(ctx context.Context, path string) (backend.Storage, error)
 	return l.New(ctx, path)
 }
 
+// Clone is a no-op.
 func (l loader) Clone(ctx context.Context, repo, path string) (backend.Storage, error) {
 	return l.New(ctx, path)
 }
 
+// Handles returns true if the given path is supported by this backend. Will always return
+// true if the directory exists.
 func (l loader) Handles(ctx context.Context, path string) error {
 	if fsutil.IsDir(path) {
 		return nil
 	}
 	return fmt.Errorf("dir not found")
 }
 
+// Priority returns the priority of this backend. Should always be higher than
+// the more specific ones, e.g. gitfs.
 func (l loader) Priority() int {
 	return 50
 }
+
 func (l loader) String() string {
```

Comment:
Exported functions should be commented, no?

Reply:
They implement an interface, so IIRC that's an exception.